uplink/src/base/serializer/mod.rs

```diff
@@ -224,23 +263,24 @@ impl<C: MqttClient> Serializer<C> {
     /// Write new data to disk until back pressure due to slow n/w is resolved
     // TODO: Handle errors. Don't return errors
     async fn slow(&mut self, publish: Publish) -> Result<Status, Error> {
-        let mut interval = time::interval(METRICS_INTERVAL);
+        let mut interval = interval(METRICS_INTERVAL);
         // Reactlabs setup processes logs generated by uplink
         info!("Switching to slow eventloop mode!!");
         self.metrics.set_mode("slow");
 
         // Note: self.client.publish() is executing code before await point
         // in publish method every time. Verify this behaviour later
-        let payload = &publish.payload[..];
-        let publish = self.client.publish(&publish.topic, QoS::AtLeastOnce, false, payload);
+        let payload = Bytes::copy_from_slice(&publish.payload[..]);
+        let publish = send_publish(self.client.clone(), publish.topic, payload);
```

Comment:
We shouldn't clone clients without out understanding what channel `tx` in `client` does on clone. In some implementations, it can result in indefinite capacity increase

Reply:
In the case of flume we are merely cloning the handle to the channel, [the channel is bounded anyways and will not increase allocation/capacity on clone](https://docs.rs/flume/0.10.14/flume/struct.Sender.html#method.clone).